tasks/computer-vision/object-detection/README.md

```diff
@@ -82,13 +82,42 @@ Detect object in given image in 80 different categories.
 - wine glass
 - zebra
 
+The data folder contains the labels for the openimages, voc and coco labels. Custom datasets of choice can also be used simply by following the format provided in the data files
+Also included is the option of using Spatial Pyramid pooling as in yolov3-spp simply by setting the parameter to 1
+
 ### Running Example
 - Navigate to tasks/computer-vision/object-detection/ directory within the repository.
-- Place the image in same direcotry for object detection and issue following command:
+- Place the image/video in same data directory for object detection and issue any of the following commands:
+
+For image data run:
 
-    `mlflow run . -P photo_name=zebra.jpg`
```

Comment:
Got error while running the first example. Please add if any prerequisite.
```
Downloading models/yolov3.weights ...
Traceback (most recent call last):
  File "yolov3_weights_to_keras.py", line 283, in <module>
    yolo.download_weights(darknet_model_path, url)
  File "yolov3_weights_to_keras.py", line 268, in download_weights
    download_if_not_exists(darknet_model_path, url)
  File "/home/user/Downloads/ai-platform/tasks/computer-vision/object-detection/download_file.py", line 30, in download_if_not_exists
    download_file(filename, url)
  File "/home/user/Downloads/ai-platform/tasks/computer-vision/object-detection/download_file.py", line 12, in download_file
    response = requests.get(url, stream=True)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/requests/sessions.py", line 519, in request
    prep = self.prepare_request(req)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/requests/sessions.py", line 462, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/requests/models.py", line 313, in prepare
    self.prepare_url(url, params)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/requests/models.py", line 387, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '--keras_model_path': No schema supplied. Perhaps you meant http://--keras_model_path?
Traceback (most recent call last):
  File "main.py", line 89, in <module>
    size = sys.argv[6], file_name = sys.argv[7], darknet_url = sys.argv[8])
  File "main.py", line 75, in workflow
    _get_or_run("yolov3_weights_to_keras", parameters, git_commit)
  File "main.py", line 64, in _get_or_run
    submitted_run = mlflow.run(".", entrypoint, parameters=parameters, use_conda=False)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/mlflow/projects/__init__.py", line 232, in run
    _wait_for(submitted_run_obj)
  File "/home/user/miniconda3/envs/mlflow-9dc31bbece1820a5bcfc860246f686839eb684f6/lib/python3.6/site-packages/mlflow/projects/__init__.py", line 249, in _wait_for
    raise ExecutionException("Run (ID '%s') failed" % run_id)
mlflow.exceptions.ExecutionException: Run (ID '50aa4490820d4040b9ed895e46227247') failed
2019/07/17 20:26:20 ERROR mlflow.cli: === Run (ID '75b4f2883a5c4f118a6f6df8db8a4526') failed ===

```

Reply:
Hello 
Will clone and see where the problem is from ASAP and get back to you.

Reply:
Fixed it, Was a little omission in the mlproject file. I have committed the changes. Didn't notice earlier because I already had the darknet weights downloaded. Works now.